src/components/main/flight/FlightInfoScreen.js

```diff
@@ -1,83 +1,65 @@
-import React from 'react'
+import React from "react";
 
 import {
   View,
   Text,
   StyleSheet,
-  Dimensions,
+  Switch,
   TouchableOpacity,
-  ActivityIndicator,
-} from 'react-native'
+  ImageBackground,
+  SafeAreaView,
+} from "react-native";
 
-import { Ionicons, FontAwesome, Feather } from '@expo/vector-icons';
+import { verticalScale, moderateScale } from "react-native-size-matters";
+import { Ionicons } from "@expo/vector-icons";
+import * as CONSTANTS from "../../utilities/Constants.js";
+import COLORS from "../../../assets/Colors.js";
+import Auth from "@aws-amplify/auth";
+import MenuBar from "../MenuBar.js";
+import i18n from "i18n-js";
 
-import Dash from 'react-native-dash';
-
-import COLORS from '../../../assets/Colors.js';
-
-import MenuBar from '../MenuBar.js';
-
-import Auth from '@aws-amplify/auth';
-
-import i18n from 'i18n-js'
-
-console.log(process.env.REACT_APP_API_KEY)
+const background_image = require("../../../assets/background/flightInfo/bg_flightInfo.png");
+const plane_image = require("../../../assets/icons/ic_plane.png");
 
 export default class FlightInfoScreen extends React.Component {
   state = {
-    isAuthenticated: 'false',
-    iata: '',
+    iata: "",
     isReady: false,
-    seatIndex: 'Economy',
-    tripIndex: 1,
-  }
+    seatIndex: "Economy",
+    tripIndex: false,
```

Comment:
if you're switching to boolean, please rename the variable to something that fits it. such as `isRoundTrip`

Reply:
Space is on purpose to keep spacing even throughout. No flex means that space has to be there. Otherwise it looks trashy and unlike the design.

Reply:
As for those margins, I balanced them out as mentioned in a PR review item to prevent the overlapping issue. I feel like this review comment is more nitty gritty than productive because even if I bring up that margin to RE things change in scale, and wrapping. There is no safe view for android and I added extra margin because your Phone is only 1 variable. Let's move on from this?

Reply:
The border got removed when I reverted commits. That is resolved.

Reply:
Spacing doesn't have to be even, it just has to look good, and the wireframe shows that a smaller gap does look good while also helping keep the button from cutting off.

before it was little to no margin at the top, now it's too much. See if you can do it inbetween. Otherwise the receipt looks unbalanced and in the flight info screen the button is cut off

Reply:
Also remember to change that variable name

Reply:
Okay, you have me confused which screen is for which?

Reply:
I am looking at multiple devices where that spacing is like prime. I do not know why yours is not. But I move that top margin up it looks bad on like 4-5 other devices.